### ISSUE TITLE
Question about learning rate

### ISSUE THREAD

**Issue by serenaTangarfff** [werner-duvaud/muzero-general#47]
Hi, 
I trained cart pole and titactoe. I used tensorboard to analysis result then found that learning rate for the cart pole changed over the training process, but the learning rate for titactoe learning remain unchanged during all steps. Could you tell me why the learning remain unchanged during all steps? Thanks

<img width="421" alt="屏幕快照 2020-05-08 上午11 30 51" src="https://user-images.githubusercontent.com/53207897/81360759-01004f80-9120-11ea-9718-6becb19e4202.png">
<img width="405" alt="屏幕快照 2020-05-08 上午11 30 39" src="https://user-images.githubusercontent.com/53207897/81360771-08bff400-9120-11ea-9023-409fa2c356be.png">

**Comment by AdrianAcala** [comment]
@serenaTangarfff , that's because of the learning rate decay.

In [cartpole](https://github.com/werner-duvaud/muzero-general/blob/master/games/cartpole.py#L85):
```
self.lr_decay_rate = 0.9  # Set it to 1 to use a constant learning rate
self.lr_decay_steps = 1000
```
whereas in [tic-tac-toe](https://github.com/werner-duvaud/muzero-general/blob/master/games/tictactoe.py#L84):
```
self.lr_decay_rate = 1  # Set it to 1 to use a constant learning rate
self.lr_decay_steps = 10000
```

If the `decay_rate = 1` it will not go down whereas if the `decay_rate < 1` it will decrease over how long the `lr_decay_steps` are. 

The exact formula is in [trainer.py](https://github.com/werner-duvaud/muzero-general/blob/af199b8e94f9fb19c8c328b468911f924e176c21/trainer.py#L247-L249):
```
lr = self.config.lr_init * self.config.lr_decay_rate ** (
    self.training_step / self.config.lr_decay_steps
)
```

Notice 1 to any power is just 1 and multiplying any number by 1 keeps the learning rate constant.

**Comment by serenaTangarfff** [comment]
@AdrianAcala Thanks for your answer. But I still cannot understand why in cartpole the learning rate be set to 0.9 but in tic-tac the learning rate is a constant number?

**Comment by AdrianAcala** [comment]
@serenaTangarfff, I think you're asking me why it's set the way it is. If that's so, the reason is because it's a little tricky. Imagine, you need to walk from one wall (starting point) of your house to another wall (optimal play). Also, imagine, you can only take 3 feet or 1 meter steps (associate this to learning rate). You may get close to the wall, but you will have a really hard time taking that last step. If you made your steps gradually smaller over time, you'll eventually get sufficiently close to the wall.

**Comment by serenaTangarfff** [comment]
@AdrianAcala Thanks for your explanation!
